product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/monitoring_using_pemx/index.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: "Monitoring using PEMx"
+---
+
+PEMx is an integrated monitoring and alerting solution within BigAnimal. It is specially designed to monitor the performance of the Postgres clusters. PEMx actively monitors various metrics of the Postgres clusters and triggers alert as the defined thresholds are exceeded. This solution smoothly integrates into the BigAnimal ecosystem, utilizing the vast amount of metric data that is gathered from Postgres clusters and their underlying infrastructure across all the regions. 
+
+PEMx renders insightful charts that empower users to actively monitor each metric's behavior. PEMx provides a streamlined path for users to take prompt and informed actions for the generated alerts. This cohesive monitoring and alerting system ensures comprehensive oversight and prompt response within the BigAnimal environment.
+
+Upon logging in, PEMx monitoring widgets are displayed on the Overview page on the project summary page. These widgets are available only for the deployed PostgreSQL clusters and are not available for the clusters in the provisioning process. They provide a high level key information on:
```

Comment:
We can also mentioned about Default & Compact view functionality on the Overview page.